### DISCUSSION TITLE
horqrux:  a minimalistic quantum state vector simulator

### DISCUSSION THREAD

**Discussion by dominikandreasseitz** [jax-ml/jax#20636] [Show and tell]
To anyone interested in building/ training quantum neural networks for complex tasks like partial differential equation solving in less than 800 lines of code:  have a look at `horqrux` ([link](https://github.com/pasqal-io/horqrux)) and its [docs](https://pasqal-io.github.io/horqrux/latest/). @GJBoth and I would greatly appreciate it!

**Comment by ASEM000** [comment]
Is this a harry potter reference? 🤔
